.ci_scripts/check_sidebars.py

```diff
@@ -0,0 +1,51 @@
+import json
+from pathlib import Path
+
+REPO_ROOT = Path(__file__).parents[1]
+DOCS_SIDEBAR = REPO_ROOT / "docs" / "sidebar.json"
+DOCS_DIR = REPO_ROOT / "docs"
+
+def get_all_docs_files():
+    files = []
+    for ext in (".md", ".mdx", ".jsx", ".md.tmpl"):
+        for f in DOCS_DIR.rglob(f"*{ext}"):
+            if f.stem.startswith("_"):
+                continue
+            path = str(f.relative_to(DOCS_DIR))[:-len(ext)]
+            if path.startswith("orga/minutes/"):
+                continue
+            if path.startswith("orga/funding/"):
+                continue
+            files.append(path)
+    return files
+
+def get_sidebar_files():
```

Comment:
Why isn't this function recursive since subitems also have link and id bits.

Reply:
Good point! I initially went for the bare minimum (we don't have nested categories yet), but this will make it more future proof. I also added support for some syntax we are not using yet to prevent errors down the line.